404.md

```diff
@@ -1,3 +1,11 @@
 ---
 permalink: /404.html
+layout: error
+errormessage: 🙅‍♀️ 404
+body_class: homepage
+errorhuman: "OpenSearch can't find that page. Ironic." 
 ---
+
+Perhaps you mistyped the URL or we've moved something around. Sorry 'bout that.
```

Comment:
This seems a little to cute to me, even though its consistent with the project website. I also don't love that we start by assuming an error on the user's part.

Maybe we should go more boilerplate?

```suggestion
We can't find the page you're looking for. Sorry bout' that. Need help still? Try our [forum](https://forum.opensearch.org/).
```